internationalization/app/es.json

```diff
@@ -253,6 +261,8 @@
           "Total XP": "PE Totales",
           "Credits": "Créditos",
           "Encumbrance": "Impedimento"
```

Comment:
Hey, looking great! Before I merge this, you'll need to add a comma here or it'll break the translations